charts/greptimedb-cluster/values.yaml

```diff
@@ -1137,7 +1137,7 @@ grafana:
     # -- The grafana image repository.
     repository: grafana/grafana
     # -- The grafana image tag.
-    tag: 11.1.3
+    tag: 11.6.0
```

Comment:
We need to modify the grafana charts dependency version to 8.12.1：https://github.com/GreptimeTeam/helm-charts/blob/main/charts/greptimedb-cluster/Chart.lock#L4

https://github.com/grafana/helm-charts/blob/main/charts/grafana/Chart.yaml